content/java/concepts/strings/terms/toCharArray/toCharArray.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.toCharArray()'
+Description: 'Returns an new character array from a given string.'
+Subjects:
+  - 'Java: Strings'
```

Comment:
```suggestion
  - 'Computer Science'
```

For reference, you can check out the [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) file for future entry's subject field metadata 🙂